airflow-code-editor/airflow_code_editor/app_builder_view.py

```diff
@@ -63,6 +62,11 @@ def repo(self, path=None):
         def save(self, path=None):
             return self._save(path)
 
+        @expose("/uploads/<path:path>", methods=["POST"])
```

Comment:
Separate out uploads path from editor save path, versioning only affects later not former.